doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -13,45 +13,46 @@ background and sends commands to this service.
 
 Launcher options
 ----------------
-The following examples show different ways that you can launch Fluent locally.
-For more information, see the description for the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
-method.
+The following examples shows different ways that you can launch Fluent locally.
+For more information, see :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`.
 
 Solver mode
 ~~~~~~~~~~~
-This example shows how you can start Fluent in solution mode:
+This example shows how you launch Fluent in solution mode:
```

Comment:
Again, it seems that we should be using "solver mode" rather than "solution" mode.